modules/490-virtualization/api/v1alpha1/clustervirtualmachineimage_types.go

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
These types can be used anywhere to interact with virtualization resources, currently they are used only by deckhouse hooks, but they are not hooks.
Currently we have no any generated APIs in deckhouse so this is precedent. I beleeve that storing them in a root is a correct choice because API's are traditionally stored in `pkg/api` directory for every go package.

Reply:
fixed in 2a67c9d2e9c926b77be5859391fe3e596e372d4e